Doc 1300/vop 002 verification of payee setting for bulk payments (#421)
swan-io/docs: docs/preview/vop/bulk-credit-transfers.mdx

```graphql {5}
mutation UpdateAccountSettings {
  updateAccountSettings(
    input: {
      accountId: "$YOUR_ACCOUNT_ID"
      allowBulkCreditTransfersWithoutBeneficiaryVerification: false
      consentRedirectUrl: "https://mycompany.com/consent"
    }
  ) {
    ... on UpdateAccountSettingsSuccess {
      account {
        settings {
          allowBulkCreditTransfersWithoutBeneficiaryVerification
        }
      }
    }
    ... on UpdateAccountSettingsConsentPending {
      __typename
      consent {
        consentUrl
      }
    }
    ... on AccountNotFoundRejection {
      message
    }
    ... on ForbiddenRejection {
      message
    }
    ... on AccountHolderNotFoundRejection {
      message
    }
  }
}
```

### Parameters {#parameters}

- `accountId`: The company account ID to update.
- `allowBulkCreditTransfersWithoutBeneficiaryVerification`: 
  - `true` = VoP optional for bulk transfers (opt out).
  - `false` = VoP required for bulk transfers (opt in).
- `consentRedirectUrl`: Required for the consent flow when opting out.

### Success response {#success-response}

> For opt-in requests (no consent required):

Reviewer: ```suggestion
For opt-in requests (no consent required).
```

```json title="Payload" {7}
{
  "data": {
    "updateAccountSettings": {
      "__typename": "UpdateAccountSettingsSuccess",
      "account": {
        "settings": {
          "allowBulkCreditTransfersWithoutBeneficiaryVerification": false
        }
      }
    }
  }
}
```

> For opt-out requests (consent required):

Reviewer: ```suggestion
For opt-out requests (consent required).
```

```json title="Payload" {4,6}
{
  "data": {
    "updateAccountSettings": {
      "__typename": "UpdateAccountSettingsConsentPending",
      "consent": {
        "consentUrl": "https://identity.swan.io/consent?consentId=$YOUR_CONSENT_ID&env=Sandbox"
      }
    }
  }
}
```

### Error handling {#error-handling}

> The mutation can return several rejection types:

Reviewer: ```suggestion
The mutation can return several rejection types.
```

| Rejection | Description |
| --- | --- |

> | **AccountNotFoundRejection** | The provided account ID doesn't exist |

Reviewer: ```suggestion
| **AccountNotFoundRejection** | The provided account ID doesn't exist. |
```

> | **AccountHolderNotFoundRejection** | The account holder doesn't exist |

Reviewer: All of these need full stops. 
```suggestion
| **AccountHolderNotFoundRejection** | The account holder doesn't exist. |
```

| **AccountHolderTypeNotEligibleRejection** | Only company account holders can modify this setting |
| **AccountMembershipNotEligibleRejection** | User lacks required permissions (`canViewAccount` + `canInitiatePayments`) |
| **AccountMembershipStatusNotEligibleRejection** | User's account membership must be **Enabled** |
| **ForbiddenRejection** | User not allowed or wrong context used |

## Sandbox testing {#sandbox-testing}

> <FieldRequirementsLegend />

Reviewer: ```suggestion
<Fieldrequirementslegend />
```

Reviewer: Sentence case

> ### Default sandbox and live requirements
> 
> | Account type | Default token requirement |
> | --- | :---: |
> | **Company** | <Opt /> |
> | **Individual** | <Req /> |
> 
> Company accounts can modify their VoP requirements through the [account settings](#updating-settings), while individual accounts always require VoP for bulk transfers.

Author: ```suggestion
### Sandbox testing

VoP requirements in sandbox match the live environment behavior. Company accounts default to optional VoP but can modify settings via the [account settings mutation](#updating-settings), while individual accounts always require VoP tokens for bulk transfers.

**Test different scenarios:**
- Company account opting in/out of mandatory VoP
- Bulk transfers with and without verification tokens
- Different VoP results using [sandbox name patterns](/preview/vop/guide-verify-beneficiary#sandbox-testing)
```

## Dispute protection {#dispute-protection}

> Swan maintains a complete audit trail of VoP setting changes and verification history for each bulk credit transfer. While this history isn't currently exposed through the API, we recommend partners log account setting changes in their own systems for reference during dispute resolution. If an end-user disputes that funds were transferred to the wrong beneficiary, this history provides essential evidence for liability determination.

Author: ```suggestion
Swan logs all VoP setting changes and verification history for dispute protection. These logs aren't currently available through the API, so we recommend you record these changes in your own systems. This audit trail helps determine liability if an end-user disputes a payment to the wrong beneficiary.
```

> Swan maintains a complete audit trail of VoP setting changes and verification history for each bulk credit transfer. While this history isn't currently exposed through the API, we recommend partners log account setting changes in their own systems for reference during dispute resolution. If an end-user disputes that funds were transferred to the wrong beneficiary, this history provides essential evidence for liability determination.

Author: My understanding is this is only a temporary fix, and the "while" hints at "until we provide it".
